qutip/tests/core/test_coefficient.py

```diff
@@ -46,6 +53,23 @@ def g(t, args):
     return np.cos(args["w"] * t * np.pi)
 
 
+def h(t, args):
+    return args["a"] + args["b"] + t
+
+
+def _assert_eq_over_interval(coeff1, coeff2, rtol=1e-12, inside=False):
+    "assert coeff1 == coeff2"
+    "inside refer to the range covered by tlistlog: [0.01, 1]"
```

Comment:
Belated review: Probably these lines should be
```python
def _assert_eq_over_interval(...):
    """ assert coeff1 == coeff2
         inside refer to the range covered by tlistlog: [0.01, 1]
    """
```
or
```python
def _assert_eq_over_interval(...):
    """ assert coeff1 == coeff2 """
    # inside refer to the range covered by tlistlog: [0.01, 1]
```
As things currently stand the docstring is `"assert coeff1 == coeff2"` and then the `"inside ..."` is just a statement on a line by itself.